source/frontend/src/features/mapSideBar/acquisition/tabs/expropriation/form8/details/ExpropriationForm8Details.tsx

```diff
@@ -0,0 +1,200 @@
+import { Col, Row } from 'react-bootstrap';
+import { FaExternalLinkAlt, FaTrash } from 'react-icons/fa';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { StyledRemoveLinkButton } from '@/components/common/buttons';
+import EditButton from '@/components/common/EditButton';
+import { Section } from '@/components/common/Section/Section';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { H3 } from '@/components/common/styles';
+import { StyledLink } from '@/components/maps/leaflet/LayerPopup/styles';
+import { DetailAcquisitionFileOwner } from '@/features/mapSideBar/acquisition/models/DetailAcquisitionFileOwner';
+import { getDeleteModalProps, useModalContext } from '@/hooks/useModalContext';
+import { Api_ExpropriationPayment } from '@/models/api/ExpropriationPayment';
+import { Api_InterestHolder } from '@/models/api/InterestHolder';
+import { formatMoney } from '@/utils/numberFormatUtils';
+import { formatApiPersonNames } from '@/utils/personUtils';
+
+import ExpropriationPaymentItemsTable from './ExpropriationPaymentItemsTable';
+
+export interface IExpropriationForm8DetailsProps {
+  form8Index: number;
+  form8: Api_ExpropriationPayment;
+  onDelete: (form8Id: number) => void;
+}
+
+export const ExpropriationForm8Details: React.FunctionComponent<
+  IExpropriationForm8DetailsProps
+> = ({ form8, form8Index, onDelete }) => {
+  const history = useHistory();
+  const match = useRouteMatch();
+  const { setModalContent, setDisplayModal } = useModalContext();
+
+  const expropiationPayeeOwner = form8.acquisitionOwnerId
+    ? DetailAcquisitionFileOwner.fromApi(form8.acquisitionOwner!)
```

Comment:
same comment about avoiding ! probably what we should do here is skip doing the conversion and just returning null if the owner isn't set.

Reply:
updated